backend/layers/business/business.py

```diff
@@ -374,13 +468,32 @@ def create_empty_dataset(self, collection_version_id: CollectionVersionId) -> Tu
 
         return (new_dataset_version.version_id, new_dataset_version.dataset_id)
 
+    def create_empty_dataset_version_for_current_dataset(
+        self, collection_version_id: CollectionVersionId, current_dataset_version_id: DatasetVersionId
+    ) -> Tuple[DatasetVersionId, DatasetId]:
```

Comment:
I don't see any functions using DatasetId. Do we need to return it? We could just return the a `DatasetVersion` and parse `DatasetId`. If needed.